.vscode/extensions.json

```diff
@@ -1,3 +1,8 @@
 {
-  "recommendations": ["astro-build.astro-vscode", "esbenp.prettier-vscode"]
+  "recommendations": [
+    "astro-build.astro-vscode",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "joshbolduc.commitlint"
```

Comment:
this extension doesn't seem to be working for me, maybe it can be removed or fixed?